test/e2e/load.go

```diff
@@ -94,17 +81,45 @@ var _ = Describe("Load capacity", func() {
 
 	type Load struct {
 		podsPerNode int
+		image       string
+		command     []string
+		skip        bool
+		deleteNs    bool
 	}
 
 	loadTests := []Load{
-		{podsPerNode: 30},
+		// The container will consume 1 cpu and 512mb of memory.
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
+		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:1.1", skip: true, deleteNs: true},
 	}
 
 	for _, testArg := range loadTests {
-		name := fmt.Sprintf("[Skipped] should be able to handle %v pods per node", testArg.podsPerNode)
+		name := fmt.Sprintf("should be able to handle %v pods per node", testArg.podsPerNode)
+		if testArg.podsPerNode == 30 {
```

Comment:
I've talked with @wojtek-t and we decided to include this new test to performance suite.